eng/Version.Details.xml

```diff
@@ -0,0 +1,20 @@
+<Dependencies>
+  <ProductDependencies>
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="6.0.100-preview.3.21202.5" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+      <Uri>https://github.com/dotnet/installer</Uri>
+      <Sha>aee38a6dd446b512b1ae510d80d2ed1c1f24e79a</Sha>
+    </Dependency>
```

Comment:
Is this one configured with the "coherent dependency" feature?

We don't want them to bump to a newer dotnet/installer than what Android or iOS has. But it might complicate things because both iOS & Android could have a different dotnet/installer.

Reply:
Yeah, the `CoherentParentDependency` attribute here should ensure that the `Microsoft.Dotnet.Sdk.Internal` dependency never moves ahead of what is in Android.